docs/source/migrating.rst

```diff
@@ -0,0 +1,189 @@
+Migrating from IPython
+======================
+
+`The Big Split <https://blog.jupyter.org/2015/04/15/the-big-split/>`__
+has changed a few things, moving the various language-agnostict
+components of IPython under the Jupyter umbrella. This doc should help
+you find what's changed, and how to keep doing what you've been doing.
+
+Where have all my files gone?
```

Comment:
While I know we are in a reactive mode right now, I think the document will be more useful if it is structured as a proactive guide on the steps a user has to take to migrate everything, such as:

1. Update to a Jupyter based install
2. Run `jupyter migrate`
3. Move config files over...
4. etc.